server/models/questions/getReaderQuestions.js

```diff
@@ -0,0 +1,135 @@
+const { pool } = require('../database')
```

Comment:
This will be a redundant code as it is exactly same as getEditorQuestions exept for `access`. So it is better to use the same model for both endpoints eg. The model `getEditorQuestions` could be named `getEditorReaderQuestions` and take in `access` as function parameter as `read` or `write`

Reply:
Eg. call `getEditorReaderQuestions({access: 'write', ...})` from `GET /questions/editor_questions` and so on

Reply:
Yeah, this seems better.